spec/ruby/core/method/compose_spec.rb

```diff
@@ -39,7 +39,7 @@ def double.call(n); n * 2; end
         double = proc { |x| x + x }
 
         (pow_2 << double).is_a?(Proc).should == true
-        (pow_2 << double).lambda?.should == true
+        (pow_2 >> double).lambda?.should == true
```

Comment:
I'm unsure if this change will be backported or not.  You may want to use `ruby_version_is` guard.

Reply:
Thanks for pointing this out, I cheated here since this spec is nested under the specs for `<<` -- I shouldn't use `>>` here .